core/extension/gdextension_interface.h

```diff
@@ -1383,6 +1384,18 @@ typedef GDExtensionVariantFromTypeConstructorFunc (*GDExtensionInterfaceGetVaria
  */
 typedef GDExtensionTypeFromVariantConstructorFunc (*GDExtensionInterfaceGetVariantToTypeConstructor)(GDExtensionVariantType p_type);
 
+/**
+ * @name get_variant_get_internal_ptr_func
+ * @since 4.4
+ *
+ * Gets a pointer to a function that can get a variant's internal value.
+ *
+ * @param p_type The Variant type.
+ *
+ * @return A pointer to a function that, when called, returns a pointer to the internal value of a variant. Check the implementation of this function (gdextension_variant_get_internal_ptr) for pointee type info of each variant type.
```

Comment:
The function is no longer named `gdextension_variant_get_internal_ptr`. If we go with my suggestion above, I guess it'd become:

```suggestion
 * @return A pointer to a function that, when called, returns a pointer to the internal value of a variant. Check the implementation of this function (gdextension_variant_get_ptr_internal) for pointee type info of each variant type.
```